pkgs/by-name/wa/warp-terminal/package.nix

```diff
@@ -99,7 +99,7 @@ meta = with lib; {
   homepage = "https://www.warp.dev";
   license = licenses.unfree;
   sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
-  maintainers = with maintainers; [ emilytrau Enzime imadnyc donteatoreo ];
```

Comment:
Thanks for your service, could you add me to `warp-terminal`'s maintainers list (to not create another PR)?

Reply:
Done :+1: